workflows/diagnostics/fv3net/diagnostics/offline/compute.py

```diff
@@ -308,7 +322,7 @@ def main(args):
     with fsspec.open(args.data_yaml, "r") as f:
         as_dict = yaml.safe_load(f)
     config = loaders.BatchesLoader.from_dict(as_dict)
-    evaluation_grid = load_grid_info(args.evaluation_grid)
+    evaluation_grid = load_grid_info(args.evaluation_grid, args.catalog_path)
```

Comment:
Could you instead load the catalog just once in the `main` func (can add this above this line)? Then instead of using the catalog path as an arg to the functions `load_grid_info, get_prediciton, etc`, pass the catalog itself.